doc/_static/style.css

```diff
@@ -12,10 +12,23 @@ h1, h2, h3, h4 {
     font-family: "Atkinson Hyperlegible", sans-serif;
 }
 
+/* Set the header font sizes */
 h1 {
     font-size: 200%;
 }
 
+h2 {
+    font-size: 180%;
+}
+
+h3 {
+    font-size: 160%;
+}
+
+h4 {
+    font-size: 140%;
+}
```

Comment:
Check with others if the 20% difference between each step is large enough. A quick web search seems to suggest that browsers use a default scaling of h1: 200%, h2: 150%, h3: 117%, h4: 100% (rather large steps). Did you try other increments?

Reply:
Yeah, I tried a few options but nothing super systematic. From trial and error, it seemed that 130% for h4 is the minimum that makes it obvious that it's a header. IMO h1 already looked good at 200%. So we could increase the spacing to from 130% to 200% or increase the maximum size past 200% to increase step size.

Reply:
Do you think it's better to have a larger step between h2 and h3, which may be commonly used? h1 is only used once and h4 is used a few times for each chapter. e.g., h2: 185%, h3: 150%, h4: 130%?

See what I see on Chrome below. It seems that **Providing Feedback** (h2) is not well separated from its h3 subtitles (e.g., **Reporting a Bug**).
![](https://user-images.githubusercontent.com/50591376/125910433-bf415e37-9f39-42d3-a301-903db94db013.png)

Reply:
Good idea, implemented in https://github.com/GenericMappingTools/pygmt/pull/1385/commits/1fbb9061068ac8f28a21bc4d146c3dfa78f81152.